InformationScripting/src/commands/CScript.cpp

```diff
@@ -73,149 +74,19 @@ Interaction::CommandResult* CScript::execute(Visualization::Item* source, Visual
-			auto toBaseQuery = query("toClass", node, args);
-			auto compositeQuery = new CompositeQuery();
-			compositeQuery->connectQuery(classesQuery, filterQuery);
-			compositeQuery->connectQuery(filterQuery, methodsOfQuery);
-			compositeQuery->connectQuery(methodsOfQuery, methodsFilter);
-			compositeQuery->connectQuery(methodsFilter, toBaseQuery);
-			compositeQuery->connectToOutput(toBaseQuery);
-			QueryExecutor queryExecutor(compositeQuery);
-			queryExecutor.execute();
-		}
-		else if (command == "callgraph")
-		{
-			// "<callgraph>"
-			auto callgraph = query("callgraph", node, args);
-			auto compositeQuery = new CompositeQuery();
-			compositeQuery->connectToOutput(callgraph);
-			QueryExecutor queryExecutor(compositeQuery);
-			queryExecutor.execute();
-		}
-		else if (command == "query19")
-		{
-			// Find all methods that are not called transitively from the TARGET method.
-
-			// $<"<methods -s=g>" - {<$<"<callgraph>" | "<addASTProperties>">$>}>$
-			// or:
-			// $<{<"<methods -s=g>", $<"<callgraph>" | "<addASTProperties>">$>}->$
-			auto allMethodsQuery = query("methods", node, QStringList("-s=g"));
-			auto callGraphQuery = query("callgraph", node, args);
-			auto astAdder = new AddASTPropertiesAsTuples();
-			auto complement = new SubstractOperator();
-			auto compositeQuery = new CompositeQuery();
-			compositeQuery->connectQuery(callGraphQuery, astAdder);
-			compositeQuery->connectQuery(allMethodsQuery, complement);
-			compositeQuery->connectQuery(astAdder, 0, complement, 1);
-			compositeQuery->connectToOutput(complement);
-			QueryExecutor queryExecutor(compositeQuery);
-			queryExecutor.execute();
-		}
-		else if (command == "color")
-		{
-			// $<"<methods>" | {<$<"<filter *quotes*>" | "<color = blue>">$, $<"<filter *brackets*>" | "<color = green>">$>} U>$
-			auto colorQuotes = new NodePropertyAdder("color", QString("blue"));
-			auto colorBrackets = new NodePropertyAdder("color", QString("green"));
-
-			auto quotesFilter = query("filter", node, QStringList("*quotes*"));
-			auto bracketsFilter = query("filter", node, QStringList("*brackets*"));
-
-			auto quotes = new CompositeQuery();
-			quotes->connectInput(0, quotesFilter);
-			quotes->connectQuery(quotesFilter, colorQuotes);
-			quotes->connectToOutput(colorQuotes);
-
-			auto brackets = new CompositeQuery();
-			brackets->connectInput(0, bracketsFilter);
-			brackets->connectQuery(bracketsFilter, colorBrackets);
-			brackets->connectToOutput(colorBrackets);
-
-			auto methods = query("methods", node, QStringList());
-			auto unionOp = new UnionOperator();
-
-			auto composite = new CompositeQuery();
-
-			composite->connectQuery(methods, quotes);
-			composite->connectQuery(methods, brackets);
-
-			composite->connectQuery(quotes, unionOp);
-			composite->connectQuery(brackets, 0, unionOp, 1);
-
-			composite->connectToOutput(unionOp);
-
-			QueryExecutor queryExecutor(composite);
-			queryExecutor.execute();
-		}
+		auto queryExecutor = new QueryExecutor();
```

Comment:
Well I was to tired to port them to the new API. In fact they were testing so they shouldn't live in this file anyways. To execute them one can still copy the  the actual script from somewhere (e.g. https://docs.google.com/document/d/11XOBGknfOR2A9kliF4os0dbqqOvrzj6g632BCkniPsI) and execute it.